services/documentation_service.py

```diff
@@ -147,6 +151,20 @@ def get_repo(self, repo_id) -> RepoFormatted:
         repo_formatted = self._format_repo(repo)
         return repo_formatted
 
+    def get_user_repo(self, user_id, repo_id) -> RepoFormatted:
```

Comment:
**curious**: since we don't really have any business logic happening here, I wonder if we can simplify `get_repo`, `get_repos`, `get_user_repo`, `get_user_repos`, by moving this formatting logic into the appropriate routers or the data service.

Reply:
Good point, removed those methods and reworked into routers.